typewritten.zsh

```diff
@@ -30,7 +30,7 @@ _set_left_prompt() {
   local return_code="%(?,,%F{red}%? )"
   if [ "$TYPEWRITTEN_DISABLE_RETURN_CODE" = true ]; then
     return_code=""
-    prompt_color = "%(?,%F{blue})"
+    prompt_color="%(?,%F{blue})"
```

Comment:
I'm sorry if my other comment wasn't clear.

The actual value of the blue color here in zsh would be: `%F{blue}`.
The extra syntax stuff (`%(?, ...`) is actually a condition checking for a return code.

Since we don't need to check again for a return code here (because it is disabled), we can just set:
```bash
prompt_color="%F{blue}"
```

Thank you for doing this btw!